### PR TITLE
Update upstream WHO URL, fixes #253

### LINKED ISSUE #253: Use new upstream WHO URL
https://frontdoor-l4uikgap6gz3m.azurefd.net/MPX/V_MPX_VALIDATED_DAILY?&$format=csv (h/t owid/monkeypox)

Will need updates to Batch config in terraform (WHO_URL)

### PR DESCRIPTION
**Pull request by abhidg**

### CODE REVIEW

File: scripts/gh_data_update/setup.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def get_cdc_data() -> list[dict]:
 def get_who_data() -> list[dict]:
 	logging.info("Getting JSON from {WHO_DATA_JSON}")
 	with open(WHO_DATA_JSON) as fh:
-		return json.load(fh)
+		return json.load(fh)["value"]
```
Comment (jim-sheldon) [inline scripts/gh_data_update/setup.py:92]: Should be `.get()` for safe access

File: scripts/agency_ingestion/run.py [hunk 2]
```diff
@@ -172,11 +172,22 @@ def __init__(self):
...
+		last_seen_date = {}
+		for row in data:
+			if row["DATE"] > last_seen_date.get(row["COUNTRY"], ""):
+				last_seen_date[row["COUNTRY"]] = row["DATE"]
+		self.data = [
+			row for row in data if
+			last_seen_date[row["COUNTRY"]] == row["DATE"]
+		]
+
+
```
Comment (jim-sheldon) [inline scripts/agency_ingestion/run.py:190]: Should use `.get()` for safe access

**Comment by jim-sheldon** [comment]
This looks like it might be more trouble than it's worth going forward.
The URL changed to something that looks like it could change anytime (and was broken when I checked it around 10 AM EST) , the request method changed, and the data format changed.
Given uncertainty around maintenance costs, what's the benefit of doing this?

**Comment by abhidg** [comment]
Agree, this may not be time-effective in keeping updated. Next time it breaks, I think we should put a notice on the map/repo saying that we are not updating upstream sources.

File: scripts/agency_ingestion/run.py [hunk 3]
```diff
@@ -180,11 +180,11 @@ def get_data(self):
...
 		last_seen_date = {}
 		for row in data:
-			if row["DATE"] > last_seen_date.get(row["COUNTRY"], ""):
-				last_seen_date[row["COUNTRY"]] = row["DATE"]
+			if row.get("DATE", "") > last_seen_date.get(row.get("COUNTRY"), ""):
+				last_seen_date[row.get("COUNTRY")] = row.get("DATE")
 		self.data = [
 			row for row in data if
-			last_seen_date[row["COUNTRY"]] == row["DATE"]
+			last_seen_date.get(row.get("COUNTRY")) == row.get("DATE")
```
Comment (jim-sheldon) [inline scripts/agency_ingestion/run.py:187]: The inner `get` is safe, the outer `get` is not

File: scripts/agency_ingestion/run.py [hunk 4]
```diff
@@ -180,11 +180,11 @@ def get_data(self):
 		# summarise data by keeping last date
 		last_seen_date = {}
 		for row in data:
-			if row["DATE"] > last_seen_date.get(row["COUNTRY"], ""):
-				last_seen_date[row["COUNTRY"]] = row["DATE"]
+			if row.get("DATE", "") > last_seen_date.get(row.get("COUNTRY"), ""):
+				last_seen_date[row.get("COUNTRY")] = row.get("DATE")
```
Comment (jim-sheldon) [inline scripts/agency_ingestion/run.py:184]: `last_seen_date.get(row.get("COUNTRY", {}))`

File: scripts/agency_ingestion/run.py (hunk 3 above)
Comment (abhidg) [reply scripts/agency_ingestion/run.py]: Any hashable type (including None) is safe to pass to `get()`, there was another possible bug where both `get()` would default to None and test for equality, so changed one of them to return `""`

**Pull request merged**